site/content/en/template/o11y_prerequisites.md

```diff
@@ -0,0 +1,12 @@
+
+Follow the steps from the [Quickstart](../quickstart) to install Envoy Gateway and the example manifest.
```

Comment:
can separate this line from current template markdown, and create a new one for all the rest places.

Reply:
tracked in #3621

Reply:
I think we cannot do that in one PR, so we just refactor when some one is trying to update.